### PR TITLE
Create hubs_tos.md

### PR DESCRIPTION
**Pull request by mafeld**

### CODE REVIEW

File: en/hubs_tos.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+{: datetime="2022-10-11" }
+
+Hubs (the “Service”) is a new, preview service that allows you to set up and manage your own set of real-time communications platforms for virtual reality, augmented reality, desktop, laptop, or mobile. It is available as a paid subscription service.  
+
+These Terms of Service explain your rights and responsibilities when you use Hubs. By purchasing and using Hubs, you assure Mozilla that you have read these terms and agree to them. 
+
+## 1. Subscribing to Hubs
+
+### You’ll Need A Firefox Account
+A Firefox Account is required to use the Service. To create a Firefox Account, you will also need to agree to the [Terms of Service](https://www.mozilla.org/about/legal/terms/services/) and [Privacy Notice](https://www.mozilla.org/privacy/firefox/) for your Firefox Account.
```
Comment (peiying2) [inline en/hubs_tos.md:13]: lower case `account` as in `Firefox account`

File: en/hubs_tos.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+### Your Privacy
+__Privacy.__ You are responsible for the privacy of the individuals who use your Hubs. This includes compliance with all applicable privacy laws. The [Hubs Privacy Notice](https://www.mozilla.org/privacy/hubs-subscription/) explains how Mozilla receives and handles data to operate the Service.
+
+Mozilla is a global organization, and our computers and service providers’ computers may be in various countries around the world, including the United States. This means that your information and your users’ information might be processed on servers located outside of the country where you and your users live, and that country may have a different level of data protection regulation than yours. By using the Service, you assure Mozilla that you consent to this kind of transfer of users’ information and that you have obtained necessary consent from the individuals who use your Hubs.
+Your Payment
```
Comment (peiying2) [inline en/hubs_tos.md:19]: Should this be a subheading?

File: en/hubs_tos.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+__Payment.__ We offer the Service as an automatically renewing subscription service. When you sign up, you authorize us to charge the payment method you provide for the subscription fees for the first term of the Service. Your plan renews automatically at the end of each term, and you authorize us to charge the subscription fees each term. If you stop paying, we will immediately suspend your account, and we may delete it after 2 months of not paying.
+
+__14-Day Refund Policy.__ The first time you subscribe to the Service through Mozilla's website, if you cancel your account within the first 14 days, you may request a refund and Mozilla will refund your first subscription term. This offer only applies the first time you subscribe. This is in addition to any rights you may have under your country’s consumer laws.
+You May Cancel Your Subscription at Any Time
```
Comment (peiying2) [inline en/hubs_tos.md:24]: Is this a subheading too?

File: en/hubs_tos.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+When you and your users submit information to Hubs, the original creator of that information continues to own the rights to their content. You grant Mozilla a worldwide, royalty-free, perpetual, irrevocable, non-exclusive, transferable, and sublicensable license to use, copy, modify, adapt, prepare derivative works from, distribute, perform, and display that information, audio, video, images, or 3D models for the purpose of operating Hubs. You also agree that we may remove metadata associated with the information or data you submit. If you or one of your users allows remixing of a scene or avatar, you agree to license that scene or avatar under a [CC-BY 3.0](https://creativecommons.org/licenses/by/3.0/legalcode) license.
+To learn more about how Hubs operates, you can see the [source code](https://github.com/mozilla/hubs).
```
Comment (peiying2) [inline en/hubs_tos.md:33]: Add a line break between the paragraphs.

File: en/hubs_tos.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+To learn more about how Hubs operates, you can see the [source code](https://github.com/mozilla/hubs). 
+
+Any ideas, suggestions, and feedback about Hubs that you provide to us are entirely voluntary, and you agree that Mozilla may use such ideas, suggestions, and feedback without compensation or obligation to you.
+You are responsible for the information you and your users send, create, or edit using Hubs, and the consequences of sending, creating, or editing that information.
```
Comment (peiying2) [inline en/hubs_tos.md:36]: Add a line break between the paragraphs.

File: en/hubs_tos.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+
+## 3. Conditions of Use
+By using Hubs, you agree that you will comply with Mozilla’s [Conditions of Use](https://www.mozilla.org/about/legal/acceptable-use/). Mozilla reserves the right to remove any content, suspend any users, and shut down any Hub or individual room it reasonably believes has violated these conditions.
+
+__Eligibility.__ To use the paid subscription service, you must be in a country where Hubs is available. Hubs is currently available in the United States, Canada, and the United Kingdom.
+Your Use of Hubs. If you have a current, paid subscription to Hubs, Mozilla gives you permission to use it according to these Terms of Service. You may not transfer or resell the Hubs Service. A purchased license only extends to the person or entity who owns the Firefox Account associated with the Hubs instance.
```
Comment (peiying2) [inline en/hubs_tos.md:42]: Add a line break between paragraphs.

File: en/hubs_tos.md [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+
+Mozilla distributes the Hubs software under an open source license. To learn more, you can read the [license for Hubs](https://github.com/mozilla/hubs/blob/master/LICENSE) or read the [FAQ](https://www.mozilla.org/MPL/2.0/FAQ/).
+
+## 5. Services Interruption; Term; Termination
+We are continuing to develop Hubs. As a result, we plan to upgrade and change Hubs over time. To do this, we might have to temporarily suspend their service and it is not always possible for us to give notice. You will not be entitled to claim expenses or damages for such suspension or limitation of the use of Hubs.
+
+We may cut off your access to Hubs, either temporarily or permanently at any time for any reason. This includes, but is not limited to, situations where we reasonably believe: (i) you have violated these Terms (ii) you create risk or possible legal exposure for Mozilla; or (iii) providing and operating Hubs is no longer commercially viable. If possible, we will make reasonable efforts to notify you through the relevant program, either Hubs .
```
Comment (peiying2) [inline en/hubs_tos.md:56]: should there be a `;` before `(ii)`?

File: en/hubs_tos.md [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+Mozilla may update these Terms from time to time. We will post the updated Terms online. If the changes are substantive, we may announce the update through Mozilla's usual channels for such announcements such as blog posts, forums, or in the particular service itself, in this case: Hubs.
+
+Your continued use of Hubs after we post the new Terms constitutes your acceptance of the new Terms. To make your review more convenient, we will post an effective date at the top of this page.
+
+## 9. Contact Us
+For support, to provide feedback, or to report abuse of Hubs or violations of the Conditions of Use, you can email us at hubs@mozilla.com.
+
+You can see our [copyright or trademark infringement claims policy](https://www.mozilla.org/about/legal/report-infringement/).
+
+For other notices, you may email us at legal-notices@mozilla.com or write to us at Mozilla Corporation (Attn: Mozilla Legal Notices, 2 Harrison Street #175, San Francisco CA 94105).
```
Comment (peiying2) [inline en/hubs_tos.md:80]: Shall we standardize how the legal address is presented so they are consistent, at least with the new ones.

File: en/hubs_tos.md (hunk 1 above)
Comment (mafeld) [reply en/hubs_tos.md:13]: fixed

File: en/hubs_tos.md (hunk 2 above)
Comment (mafeld) [reply en/hubs_tos.md:19]: I changed this around to address a duplicate heading.

**Review by peiying2 — CHANGES_REQUESTED** [review]
A few common findings to call out, please do a find and replace globally as I didn't call them all out:
- Lower case Firefox account except for heading
- `'` (straight apostrophe) should be changed to curly ones `’`
- " (straight quote) should be changed to curly ones “ and ” (opening, closing) - the only exception is the date of the doc which remain unchanged.

Additionally,
- Do we need a version for this doc?
- Legal address: do we need to add `USA`? Also the format varies from doc to doc. Let's take the opportunity to standardize it.

File: en/hubs_tos.md (hunk 3 above)
Comment (mafeld) [reply en/hubs_tos.md:24]: yup

File: en/hubs_tos.md (hunk 4 above)
Comment (mafeld) [reply en/hubs_tos.md:33]: fixed

File: en/hubs_tos.md (hunk 5 above)
Comment (mafeld) [reply en/hubs_tos.md:36]: fixed

File: en/hubs_tos.md (hunk 6 above)
Comment (mafeld) [reply en/hubs_tos.md:42]: fixed and bolded

File: en/hubs_tos.md (hunk 7 above)
Comment (mafeld) [reply en/hubs_tos.md:56]: yup. done

File: en/hubs_tos.md (hunk 8 above)
Comment (mafeld) [reply en/hubs_tos.md:80]: yes. Going forward I'll use the same format as the subscription services TOS.

**Comment by mafeld** [comment]
@peiying2 I think I did something wrong to address the comments because merging is now blocked. I tried to edit the doc to address your comments. Do you know what the issue is?

**Comment by mafeld** [comment]
> 
>     * `'` (straight apostrophe) should be changed to curly ones `’`
> 
>     * " (straight quote) should be changed to curly ones “ and ” (opening, closing)
> 

I don't know how to insert those in markdown. no matter what I press, it gives me the straight quote.

**Comment by peiying2** [comment]
> @peiying2 I think I did something wrong to address the comments because merging is now blocked. I tried to edit the doc to address your comments. Do you know what the issue is?

If you are trying to merge it, then you can't. I added more "general" comments above. Scroll up to see my review summary. I think once you fix them all (nitpicking stuff), request me to review again. Once I approve it, you then can merge it.

**Comment by peiying2** [comment]
> > ```
> > * `'` (straight apostrophe) should be changed to curly ones `’`
> > 
> > * " (straight quote) should be changed to curly ones “ and ” (opening, closing)
> > ```
> 
> I don't know how to insert those in markdown. no matter what I press, it gives me the straight quote.

Actually it's quite inconsistent throughout. I copied them from the text.  One exception - keep the straight quotes on the dates. The curly ones only apply to text. Let me figure out the keypad combinations for these.

**Comment by peiying2** [comment]
> > @peiying2 I think I did something wrong to address the comments because merging is now blocked. I tried to edit the doc to address your comments. Do you know what the issue is?
> 
> If you are trying to merge it, then you can't. I added more "general" comments above. Scroll up to see my review summary. I think once you fix them all (nitpicking stuff), request me to review again. Once I approve it, you then can merge it.

On Mac, Option

**Comment by peiying2** [comment]
@mafeld what's the difference between the hubs PN/ToS and Mozilla Hubs PN/ToS. And now we have subscription hubs PN/ToS.

**Comment by mafeld** [comment]
The old "mozilla_hubs" tos and PN were for our previous product where everything was hosted and managed by Mozilla. 
The new "hubs" tos and PN are for the new subscription product that we're offering where the subscriber will manage their community. 
We need to keep the old "mozilla_hubs" tos and PN because we're going to keep the old Hubs around for a little while. Eventually, we'll transition off the old Hubs to avoid any confusion around the name, which remains the same.

The "subscription services" pn and TOS are for the new bundle of VPN and Relay. We will soon get rid of the individual PN and terms for VPN and Relay.

**Review by peiying2 — APPROVED** [review]
Thanks for the update. LGTM!

**Pull request merged**